app/models/manageiq/providers/redhat/infra_manager/provision.rb

```diff
@@ -9,8 +9,8 @@ def destination_type
     "Vm"
   end
 
-  def get_provider_destination
-    return nil if destination.nil?
-    destination.with_provider_object { |rhevm_vm| return rhevm_vm }
+  def with_provider_destination(&block)
+    return if destination.nil?
+    destination.with_provider_object { |obj| block.call(obj) }
```

Comment:
I think codeclimet wants you to do:
destination.with_provider_object { |obj| yield obj }
and remove the &block argument.

http://www.rubydoc.info/gems/rubocop/0.47.1/RuboCop/Cop/Performance/RedundantBlockCall